UPGRADE.md

```diff
@@ -2,6 +2,39 @@
 
 ## Version 4.* to 5.0
 
+### Dropped legacy Simple Token Parser
+
+Tokens which are not valid PHP variable names (e.g. `##0foobar##`) are not supported anymore.
+
+### Dropped $GLOBALS['TL_KEYWORDS']
+
+Keyword support in articles, and as such also `$GLOBALS['TL_KEYWORDS']`, have been removed without
+replacement.
+
+### Dropped legacy routing
+
+The legacy routing support has been dropped. As such, the `getPageIdFromUrl` and `getRootPageFromUrl`
+hooks do not exist anymore. Use Symfony routing instead.
+
+### Removed initialize.php
+
+The `initialize.php` support has been removed without replacement. Register your own Symfony
+Routes to the routing instead.
+
+### Removed ClassLoader class
+
+The `Contao\ClassLoader` has been removed without replacement. Use Composer autoloading instead.
```

Comment:
Also the `TemplateLoader` and `ModuleLoader`

Reply:
The template loader has not been removed (and cannot be removed yet).